ui/redux/selectors/collections.js

```diff
@@ -335,16 +225,27 @@ export const selectAreBuiltinCollectionsEmpty = (state: State) => {
   return !notEmpty;
 };
 
-export const selectClaimIdsForCollectionId = createSelector(selectCollectionForId, (collection) => {
-  const items = (collection && collection.items) || [];
+export const selectClaimIdsForCollectionId = createSelector(
+  selectCollectionForId,
+  selectClaimIdsByUri,
+  (collection, byUri) => {
+    const items = (collection && collection.items && collection.items.filter(Boolean)) || [];
 
-  const ids = items.map((item) => {
-    const { claimId } = parseURI(item);
-    return claimId;
-  });
+    let ids = [];
+    for (const item of items) {
+      const claimId = byUri[normalizeURI(item)];
 
-  return ids;
-});
+      if (!claimId) {
+        ids = [];
+        break;
+      }
+
+      ids.push(claimId);
+    }
```

Comment:
- This will return blank array when any entry fails.  Is that the intention here?
- Is `normalizeURI` safe without a try-catch?  e.g. another app accidentally added an entry like `lbry://hello world`